powerpacks.go

```diff
@@ -50,15 +50,31 @@ type ShortCode struct {
 	Added_on         string `json:"added_on,omitempty"`
 }
 
+type Tollfree struct {
```

Comment:
You might want to rename this to something less generic and more scoped to this package, e.g. `PowerpackTollFreeBase`